Q/QPALM/build_tarballs.jl

```diff
@@ -65,15 +39,11 @@ platforms = supported_platforms()
 
 # The products that we will ensure are always built
 products = Product[
-                   LibraryProduct("libladel", :libladel),
-                   LibraryProduct("libqpalm", :libqpalm),
+    LibraryProduct("libqpalm_jll", :libqpalm_jll),
 ]
 
 # Dependencies that must be installed before this package can be built
-dependencies = [
-    Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"))
-    Dependency(PackageSpec(name="SuiteSparse_jll", uuid="bea87d4a-7f5b-5778-9afe-8cc45184846c"))
-]
+dependencies = []
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.3")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"10.2.0")
```

Comment:
Why GCC 10?  For maximum compatibility we prefer to use the _oldest_ GCC version which is able to compile the code.

Reply:
Good point, I just selected that because it was the version I developed with. I'll remove the explicit version, it should build without issues on GCC 4.8.5.